### PR TITLE
Update 21.co.xml

### PR DESCRIPTION
**Pull request by cschanaj**
Related: https://github.com/EFForg/https-everywhere/issues/3069#issue-111248804

### CODE REVIEW

File: src/chrome/content/rules/21.co.xml [hunk 1]
```diff
@@ -1,13 +1,29 @@
 <!--
-	CDN bucket:
-		- dotco-prod.s3.amazonaws.com
+	Non-functional hosts
+		Timeout was reached:
+			 - bitnodes.21.co
+
+		4xx client error:
+			 - api.21.co
```
Comment (J0WI) [inline src/chrome/content/rules/21.co.xml]: https://api.21.co/lists/

File: src/chrome/content/rules/21.co.xml [hunk 2]
```diff
@@ -1,13 +1,29 @@
 <!--
-	CDN bucket:
-		- dotco-prod.s3.amazonaws.com
+	Non-functional hosts
+		Timeout was reached:
+			 - bitnodes.21.co
```
Comment (J0WI) [inline src/chrome/content/rules/21.co.xml]: WFM

File: src/chrome/content/rules/21.co.xml (hunk 1 above)
Comment (cschanaj) [inline src/chrome/content/rules/21.co.xml]: thanks.

File: src/chrome/content/rules/21.co.xml (hunk 2 above)
Comment (cschanaj) [inline src/chrome/content/rules/21.co.xml]: Double checked. `bitnodes` works. thanks.

**Pull request merged**